src/components/Reports/StaffExpenseReport/Tables/StaffReportTable.test.tsx

```diff
@@ -0,0 +1,183 @@
+import React from 'react';
+import { ThemeProvider } from '@mui/material/styles';
+import { LocalizationProvider } from '@mui/x-date-pickers';
+import { AdapterLuxon } from '@mui/x-date-pickers/AdapterLuxon';
+import { render, waitFor, within } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import TestRouter from '__tests__/util/TestRouter';
+import { GqlMockedProvider } from '__tests__/util/graphqlMocking';
+import theme from 'src/theme';
+import { ReportsStaffExpensesQuery } from '../GetStaffExpense.generated';
+import { TableType } from '../Helpers/StaffReportEnum';
+import { StaffReportTable, StaffReportTableProps } from './StaffReportTable';
+
+const mutationSpy = jest.fn();
+
+const router = {
+  pathname: '/accountLists/[accountListId]/reports/staffExpense',
+  query: { accountListId: 'account-list-1' },
+  isReady: true,
+};
+
+interface TestComponentProps {
+  isEmpty?: boolean;
+  tableProps?: Partial<StaffReportTableProps>;
+}
+
+const TestComponent: React.FC<TestComponentProps> = ({
+  tableProps,
+  isEmpty,
+}) => (
+  <LocalizationProvider dateAdapter={AdapterLuxon}>
```

Comment:
This provider is usually only needed when the test component renders a date picker.

```suggestion
```